.github/workflows/superset-docs-deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Docs Testing
+
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+    types: [synchronize, opened, reopened, ready_for_review]
+jobs:
+  build-deploy:
+    name: Build & Deploy
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: docs
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: recursive
+      - name: yarn install
```

Comment:
for these yarn commands, shouldn't there be a `cd docs` before?

Reply:
I believe the `working-directory` parameter does that.